[2021-04-23 17:23:22,026] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T16:00:00+00:00 [queued]>
[2021-04-23 17:23:22,093] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T16:00:00+00:00 [queued]>
[2021-04-23 17:23:22,096] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 17:23:22,097] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2021-04-23 17:23:22,099] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 17:23:22,119] {taskinstance.py:1063} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc> on 2021-04-23T16:00:00+00:00
[2021-04-23 17:23:22,128] {standard_task_runner.py:52} INFO - Started process 296 to run task
[2021-04-23 17:23:22,136] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'create_cluster_dataproc', '2021-04-23T16:00:00+00:00', '--job-id', '27', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmp5jcrzaiv', '--error-file', '/tmp/tmp22xvr7l_']
[2021-04-23 17:23:22,141] {standard_task_runner.py:77} INFO - Job 27: Subtask create_cluster_dataproc
[2021-04-23 17:23:22,238] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T16:00:00+00:00 [running]> on host 760ffd3d790b
[2021-04-23 17:23:22,344] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-23T16:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-23T16:00:00+00:00
[2021-04-23 17:23:22,348] {dataproc.py:603} INFO - Creating cluster: ephemeral-recent-played-spark-cluster-20210423
[2021-04-23 17:27:29,523] {dataproc.py:547} INFO - Cluster created.
[2021-04-23 17:27:29,826] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=create_cluster_dataproc, execution_date=20210423T160000, start_date=20210423T172322, end_date=20210423T172729
[2021-04-23 17:27:30,195] {local_task_job.py:146} INFO - Task exited with return code 1
